features/articles_belong_to_categories.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Article can be linked to a category
+  As an owner
+  In order to keep the site organized and easily navigated
+  I would like to be able to categorize articles
+
+  Background:
+    Given the following categories exist
+      | name    |
+      | Fashion |
+
+  Scenario: User can categorize an article
+    Given I am on the "Create Article" page
+    And show me the page
+    When I select "Fashion" from "categories"
+    And I fill in "Title" with "Holger is the best"
+    And I fill in "Content" with "Dreamteam for the win!"
+    And I click on "Create Article"
+    Then I should see "Article successfully created"
+    And I should see "Fashion"
+    And I am on the "Index" page
```

Comment:
` And I am on the "Index" page` seems like you are telling Capybara to visit a page, not asserting that you are on a page. Is it an assertion or is it a command?

Reply:
We want it to visit the page! :) 
We wanted to see if the category was displayed on the index page.
It's within scope & we thought that a new Scenario was overkill, so we just added two lines to this Scenario. 
But I can see that we might want to add a Scenario?! 

what do you think?

Reply:
I see. Thx for the clarification. A bit tricky. At the very least I would change `And` to `When` and perhaps rephrase to `When I visit the "Index" page` to be more clear. 

Also, hold back on the exclamation marks in your comments @holgertidemand They can be easily misunderstood.

Reply:
We will refactor this! And no more '!' :)